locales/en.json

```diff
@@ -114,6 +114,7 @@
     "ip6tables_unavailable": "You cannot play with ip6tables here. You are either in a container or your kernel does not support it.",
     "iptables_unavailable": "You cannot play with iptables here. You are either in a container or your kernel does not support it.",
     "ldap_initialized": "LDAP has been initialized",
+    "ldap_init_failed_to_create_admin": "LDAP initialization failed to create admin. Aborting.",
```

Comment:
1/  "LDAP initialization failed to create admin **user**."  ?
2/ Remove the "Aborting" and use `installation_failed`string ?